runelite-client/src/main/java/net/runelite/client/plugins/hiscore/HiscoreConfig.java

```diff
@@ -64,6 +70,17 @@ default boolean virtualLevels()
 		return true;
 	}
 
+	@ConfigItem(
+		position = 3,
+		keyName = "post99SkillDisplayStyle",
+		name = "Maxed skills",
+		description = "How to display skills that are level 99."
+	)
+	default MaxedSkillsStyle maxedSkillsStyle()
+	{
+		return virtualLevels() ? VIRTUAL_LEVELS : SHOW_99;
```

Comment:
Can the this be reverted back into a migrate config?

Reply:
Done.